gcapi/check_version.py

```diff
@@ -0,0 +1,41 @@
+import warnings
+from importlib.metadata import version as get_version
+
+import httpx
+from packaging import version
+
+
+class UnsupportedVersionError(Exception):
+    pass
+
+
+def check_version(base_url):
+    package_name = "gcapi"
+
+    current_version = get_version(package_name)
+    with httpx.Client() as client:
+        response = client.get(f"{base_url}/gcapi/")
+
+        json = response.json()
+        latest_version = json["latest_version"]
+        lowest_supported_version = json["lowest_supported_version"]
```

Comment:
Removes shadowing of standard library

```suggestion
    current_version = get_version(package_name)

    with httpx.Client() as client:
        response = client.get(f"{base_url}/gcapi/").json()

        api_data = response.json()

        latest_version = api_data["latest_version"]
        lowest_supported_version = api_data["lowest_supported_version"]
```